en/clinic-user-guide.md

```diff
@@ -0,0 +1,674 @@
+
+</div>
+<div label="Offline deployment">
+
+If your cluster cannot access the Internet, you can deploy Diag using the offline method.
+
+1. Download the Diag chart.
+
+    If your cluster cannot access the Internet, you cannot install Diag and other components by configuring the Helm repo. In this situation, you need to download the chart files on a machine with Internet access and then copy the file to the cluster.
+
+    To download Diag chart files, you can use the following command:
+
+    ```shell
+    wget http://charts.pingcap.org/diag-v0.7.1.tgz
+    ```
+
+    Copy `diag-v0.7.1.tgz` to the cluster and unpack it to the current directory.
+
+    ```shell
+    tar zxvf diag-v0.7.1.tgz
+    ```
+
+2. Download the Diag image.
+
+    You need to download the Diag image on a machine that has Internet access and then use the `docker load` command to load the image to the cluster.
+
+    The Diag image is `pingcap/diag:v0.7.1`. You can download and save the image using the following commands:
+
+    ```shell
+    docker pull pingcap/diag:v0.7.1
+    docker save -o diag-v0.7.1.tar pingcap/diag:v0.7.1
+    ```
+
+    Then, copy the archived image to the cluster and use the `docker load` command to load the image to the cluster:
+
+    ```shell
+    docker load -i diag-v0.7.1.tar
+    ```
+
+3. Configure the `values-diag-collector.yaml` file.
+
+    Modify your access token in the `${HOME}/diag-collector/values-diag-collector.yaml` file.
+
+    Other configuration parameters such as `limits`, `requests`, and `volume` can be modified according to your needs.
+
+    > **Note:**
+    >
+    > To get the token, refer to [Step 2: Log in to the Clinic Server and get an access token](#step-2-log-in-to-the-clinic-server-and-get-an-access-token).
+
+4. Install Diag.
+
+    Install Diag using the following command:
+
+    ```shell
+    helm install diag-collector ./diag --namespace=tidb-admin
+    ```
+
+    > **Note:**
+    >
+    > The `namespace` should be the same as that of TiDB Operator. If TiDB Operator is not deployed, deploy TiDB Operator first and then deploy Diag.
+
+5. (Optional) Set a persistent volume.
+
+    This step sets a data volume for Diag to persist its data. To set the volume, you can configure the `diag.volume` field with the volume type in the `${HOME}/diag-collector/values-diag-collector.yaml` file. The following examples are PVC and Host:
+
+    ```
+    # Use PVC volume type
+    volume:
+      persistentVolumeClaim:
+        claimName: local-storage-diag
+    ```
+
+    ```
+    # Use Host volume type
+    volume:
+      hostPath:
+        path: /data/diag
+    ```
+
+    > **Note:**
+    >
+    > - Setting a volume on multiple disks is not supported.
+    > - All types of StorageClass are supported.
+
+</div>
+<div label="Least privilege deployment">
+
+> **Note:**
+>
+> Least privilege deployment is to deploy Diag to the namespace of the cluster so that Diag can collect data only in the namespace but not across namespaces.
```

Comment:
```suggestion
> Least privilege deployment is to deploy Diag to the namespace of the cluster so that Diag can collect data only in that namespace but not across namespaces.
```